src/PeoplePicker.ts

```diff
@@ -1 +0,0 @@
-export * from './components/PeoplePicker/index';
```

Comment:
Approved

Overall comments:

Nice work taking this on and a great revision overall. Some minor things on the example:

* If no results are available, have a "No suggestions" message
* DocumentCard example really painful to understand because the cards are gigantic.
* Arrow keying through code typed in moves focus to items. It would be nice to adjust the focusZone logic to respect cursor position for arrowing left/right from within an input element.
* Because the example allows you to choose the same item more than once, deleting one of the items deletes them all since they all share the same key. maybe you should use item.key + index for the key?

I think we'll keep revising this, but this is a great first step!